models/user.py

```diff
@@ -101,8 +102,14 @@ def password_match(cls, password_to_verify, values, **kwargs):
             raise ValueError('passwords do not match')
         return password_to_verify
 
+
 class user_in_db(user_to_register):
 
```

Comment:
Class description is missing!! Please add class description wherever it is missing